src/components/ListNotes/ListNotes.jsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react'
 import {v4 as uuid} from 'uuid'
 
+
 import './listnotes.css'
 import NoteCard from '../NoteCard/NoteCard'
 import { useNote } from '../../context/NoteContext'
 const ListNotes = () => {
+  
     const {notes} = useNote();
+    const pinnednotes = notes.filter((obj)=>obj.pinned===true).filter((obj)=>obj.archive===false).filter((obj)=>obj.trash===false)
+    const unpinnednotes = notes.filter((obj)=>obj.pinned===false).filter((obj)=>obj.archive===false).filter((obj)=>obj.trash===false)
```

Comment:
For readability going to keep this version, will change in further versions. I want to keep the map function clean to add drag and drop later , to avoid confusion there.